### PR TITLE
added center-method focus to sdfstudio-data

### PR DESCRIPTION
**Pull request by marcostrinca**
A quick update do sdfstudio dataparser allowing users to choose "focus" to center poses.
An example: `ns-train neus sdfstudio-data ./mydata --center-method focus`
Tested with neus and neus-facto with my own datasets after running `process_nerfstudio_to_sdfstudio.py`

I've used the code from the updated version of nerfstudio repo.

Note: the change I did in nerfstudio_collate.py to remove torch._six.string_classes was to make it compatible with new versions of pytorch. I've been using pytorch 2.0.1 by the way.

### CODE REVIEW

**Comment by marcostrinca** [comment]
Ops.
I've just realized there is another PR which implements this same feature.
Closing this one.

**Comment by marcostrinca** [comment]
After reading #90 I thought this PR was unnecessary.  Took me a couple minutes to realize the first is related to nerfstudio dataparser instead of sdfstudio dataparser.
Sorry about the mess!

**Comment by niujinshuchong** [comment]
@marcostrinca Thanks for the PR but seems like this PR contains multiple changes (centering, adding mini-omidata, foreground masks). Would you mind split it to different PRs?
